system/vmware-monitoring/templates/collector-configmap.yaml

```
@@ -176,6 +176,10 @@ data:
         key: "cpu|capacity_provisioned"
       - metric_suffix: "cpu_corecount_provisioned"
         key: "cpu|corecount_provisioned"
+      - metric_suffix: "cpu_corepeak_usage"
+        key: "cpu|peak_cpu_core_usage"      
+      - metric_suffix: "cpu_coreutilization_percentage"
+        key: "cpu|coreUtilization_average"
```

Make sure you add the unit at the end, if possible. I suppose in these examples it is percentage, no?

```suggestion
      - metric_suffix: "cpu_peak_core_usage_percentage"
        key: "cpu|peak_cpu_core_usage"      
      - metric_suffix: "cpu_core_utilization_percentage"
        key: "cpu|coreUtilization_average"
```

---

I wasn't sure about the unit in  'cpu|peak_cpu_core_usage', VROPS documentation is not clear about that. These look good to me, thanks.

---

i haven't looked it up, so you better double check ;)

---

Can spot it either, even when I am looking into an active vROps deployment. Question is, what do you expect from it, when you don't know, what it is about?

---

Hopefully a percentage, if it's MHz instead, then we can compute the percentage ourselves...